config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "isisvanderplas",
```

Comment:
get this from process.ENV.DB_USERNAME because team members have a different username in their own local computer, you also need to turn the extension from .json to .js to achive this. Also you need to set the DB_USERNAME environment variable in your ~/.bash_profile